src/jobs/jobs.controller.js

```diff
@@ -2,28 +2,30 @@ const winston = require('winston');
 const persistence = require('./jobs.persistence');
 
 /**
- * Add a new job into the system if not exists.
+ * Add a new job into the system if it does not exists.
  *
- * @param {object} offer
+ * @param {object} job
  */
-async function postJob(offer) {
-  winston.info('jobs-controller:postJob', offer);
-  const existingOffer = await persistence.getOffer(offer);
-  if (!existingOffer) {
-    return persistence.saveOffer(offer);
-  }
-  return Promise.resolve(existingOffer);
+async function postJob(job) {
+  winston.info('jobs-controller:postJob', job);
+  const existingJob = await persistence.getJob(job);
+
+  return !existingJob
```

Comment:
you can change the order of the expressions in the ternary operator and remove the negation